libminifi/include/utils/StringUtils.h

```diff
@@ -74,12 +74,10 @@ struct string_traits<wchar_t>{
 class StringUtils {
  public:
   /**
-   * Converts a string to a boolean
-   * Better handles mixed case.
+   * Checks and converts a String to a Boolean
    * @param input input string
-   * @param output output string.
+   * @returns a boolean or an empty value
```

Comment:
Might be better worded as `"an optional of a boolean: true if the string is "true" (ignoring case), false if it is "false"  (ignoring case), nullopt for any other value"`